<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureDialog</class>
 <widget class="QDialog" name="ConfigureDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>yuzu 设置</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="selectorList">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="ConfigureGeneral" name="generalTab">
        <property name="accessibleName">
         <string>通用</string>
        </property>
        <attribute name="title">
         <string>通用</string>
        </attribute>
       </widget>
       <widget class="ConfigureUi" name="uiTab">
        <property name="accessibleName">
         <string>UI</string>
        </property>
        <attribute name="title">
         <string>游戏列表</string>
        </attribute>
       </widget>
       <widget class="ConfigureSystem" name="systemTab">
        <property name="accessibleName">
         <string>系统</string>
        </property>
        <attribute name="title">
         <string>系统</string>
        </attribute>
       </widget>
       <widget class="ConfigureProfileManager" name="profileManagerTab">
        <property name="accessibleName">
         <string>简介</string>
        </property>
        <attribute name="title">
         <string>简介</string>
        </attribute>
       </widget>
       <widget class="ConfigureFilesystem" name="filesystemTab">
        <property name="accessibleName">
         <string>文件系统</string>
        </property>
        <attribute name="title">
         <string>文件系统</string>
        </attribute>
       </widget>
       <widget class="ConfigureInput" name="inputTab">
        <property name="accessibleName">
         <string>控制</string>
        </property>
        <attribute name="title">
         <string>控制</string>
        </attribute>
       </widget>
       <widget class="ConfigureHotkeys" name="hotkeysTab">
        <property name="accessibleName">
         <string>热键</string>
        </property>
        <attribute name="title">
         <string>热键</string>
        </attribute>
       </widget>
       <widget class="ConfigureCpu" name="cpuTab">
        <property name="accessibleName">
         <string>CPU</string>
        </property>
        <attribute name="title">
         <string>CPU</string>
        </attribute>
       </widget>
       <widget class="ConfigureCpuDebug" name="cpuDebugTab">
        <property name="accessibleName">
         <string>调试</string>
        </property>
        <attribute name="title">
         <string>调试</string>
        </attribute>
       </widget>
       <widget class="ConfigureGraphics" name="graphicsTab">
        <property name="accessibleName">
         <string>图形</string>
        </property>
        <attribute name="title">
         <string>图形</string>
        </attribute>
       </widget>
       <widget class="ConfigureGraphicsAdvanced" name="graphicsAdvancedTab">
        <property name="accessibleName">
         <string>高级</string>
        </property>
        <attribute name="title">
         <string>图形高级</string>
        </attribute>
       </widget>
       <widget class="ConfigureAudio" name="audioTab">
        <property name="accessibleName">
         <string>声音</string>
        </property>
        <attribute name="title">
         <string>声音</string>
        </attribute>
       </widget>
       <widget class="ConfigureDebug" name="debugTab">
        <property name="accessibleName">
         <string>调试</string>
        </property>
        <attribute name="title">
         <string>调试</string>
        </attribute>
       </widget>
       <widget class="ConfigureWeb" name="webTab">
        <property name="accessibleName">
         <string>网络</string>
        </property>
        <attribute name="title">
         <string>网络</string>
        </attribute>
       </widget>
       <widget class="ConfigureService" name="serviceTab">
        <property name="accessibleName">
         <string>服务</string>
        </property>
        <attribute name="title">
         <string>服务</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ConfigureGeneral</class>
   <extends>QWidget</extends>
   <header>configuration/configure_general.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ConfigureSystem</class>
   <extends>QWidget</extends>
   <header>configuration/configure_system.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ConfigureProfileManager</class>
   <extends>QWidget</extends>
   <header>configuration/configure_profile_manager.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ConfigureFilesystem</class>
   <extends>QWidget</extends>
   <header>configuration/configure_filesystem.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ConfigureAudio</class>
   <extends>QWidget</extends>
   <header>configuration/configure_audio.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ConfigureDebug</class>
   <extends>QWidget</extends>
   <header>configuration/configure_debug.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ConfigureCpu</class>
   <extends>QWidget</extends>
   <header>configuration/configure_cpu.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ConfigureCpuDebug</class>
   <extends>QWidget</extends>
   <header>configuration/configure_cpu_debug.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ConfigureGraphics</class>
   <extends>QWidget</extends>
   <header>configuration/configure_graphics.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ConfigureGraphicsAdvanced</class>
   <extends>QWidget</extends>
   <header>configuration/configure_graphics_advanced.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ConfigureWeb</class>
   <extends>QWidget</extends>
   <header>configuration/configure_web.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ConfigureUi</class>
   <extends>QWidget</extends>
   <header>configuration/configure_ui.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ConfigureInput</class>
   <extends>QWidget</extends>
   <header>configuration/configure_input.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ConfigureHotkeys</class>
   <extends>QWidget</extends>
   <header>configuration/configure_hotkeys.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ConfigureService</class>
   <extends>QWidget</extends>
   <header>configuration/configure_service.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigureDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigureDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
